frontend/rust-lib/flowy-database/src/services/field/type_options/date_type_option/date_type_option_entities.rs

```diff
@@ -71,36 +78,27 @@ impl ToCellChangesetString for DateCellChangeset {
   }
 }
 
-#[derive(Default, Clone, Debug)]
-pub struct DateCellData(pub Option<i64>);
-
-impl std::convert::From<DateCellData> for i64 {
-  fn from(timestamp: DateCellData) -> Self {
-    timestamp.0.unwrap_or(0)
-  }
-}
-
-impl std::convert::From<DateCellData> for Option<i64> {
-  fn from(timestamp: DateCellData) -> Self {
-    timestamp.0
-  }
+#[derive(Default, Clone, Debug, Serialize, Deserialize)]
+pub struct DateCellData {
+  pub timestamp: Option<i64>,
+  pub include_time: bool,
 }
 
 impl FromCellString for DateCellData {
   fn from_cell_str(s: &str) -> FlowyResult<Self>
   where
     Self: Sized,
   {
-    let num = s.parse::<i64>().ok();
-    Ok(DateCellData(num))
+    let result: DateCellData = serde_json::from_str(s).unwrap_or_default();
```

Comment:
It will break the changes. Because the data in disk is just a timestamp. So It can't be deserialized using the default deserializer provided by serde. We need to write our version deserializer.